### ISSUE TITLE
path names that include '[' or ']' seem to fail

### ISSUE THREAD

**Issue by disputin** [khenriks/mp3fs#71]
For example:

```-?????????  ? ?    ?             ?            ? '01_-_DNA_feat_Suzanne_Vega_-_Tom'\''s_Diner_[extended_mix].mp3'
-?????????  ? ?    ?             ?            ? '02_-_Seduction_-_Groove_Me_[cub_mix].mp3'
-?????????  ? ?    ?             ?            ? '03_-_Overweight_Pooch_-_Hip_House_Party_[extended_mix].mp3'
-rw-r--r--  1 sean sean 6847805 Nov  2 21:06 "04_-_Certain_Ratio,_A_-_Won't_Stop_Loving_You.mp3"
-?????????  ? ?    ?             ?            ? '05_-_Jazzi_P_-_Feel_The_Rhythm_[club_mix].mp3'
-?????????  ? ?    ?             ?            ? '06_-_New_Life_-_Got_2_B_Free_[italo_house_mix].mp3'
-?????????  ? ?    ?             ?            ? '07_-_Steve_Harvey_-_I'\''m_The_One_[extended_vocal_mix].mp3'
-rw-r--r--  1 sean sean 8120520 Nov  2 21:06 '08_-_La_Mix_feat_Problem_Child,_The_-_Coming_Back_For_More_(Back_Off_Girl).mp3'
-?????????  ? ?    ?             ?            ? '09_-_Jazzi_P_-_Feel_The_Rhythm_[the_tunnel_mix].mp3'
-?????????  ? ?    ?             ?            ? '10_-_Steve_Harvey_-_I'\''m_The_One_[extended_vocal_dub_mix].mp3'
```

**Comment by khenriks** [comment]
Sorry about this bug, this is a duplicate of #66. It's fixed in the latest version (1.1.1). If you upgrade to the newest version, it should be fixed.

**Comment by disputin** [comment]
Thanks. I’ll install from git instead of the Debian package

Sean
